src/plugins/Polls/schema.json

```diff
@@ -26,14 +26,23 @@
             "type": "array",
             "title": "results"
         },
-        "sender": {
+        "sender_name": {
```

Comment:
change schema is break change.

v1.16.2 include the plugin on release.

CC @guanbinrui @Jack-Works

Reply:
Agree. You need to make it back-ward compatibility or add a new version.

Reply:
i think, we can next release disable this plugin in production.

it is data structure not stable plugin.